ce/api/util.py

```diff
@@ -114,36 +114,61 @@ def mean_datetime(datetimes):
     return datetime.fromtimestamp(mean, tz=timezone.utc)
 
 
-# valid cell method parameters for the API. Add new ones here as needed.
-VALID_CELL_METHODS_PARAMETERS = ("mean", "standard_deviation", "percentile")
+# valid climatological statistic method parameters for the API. Add new ones here as needed.
+VALID_CLIM_STAT_PARAMETERS = ("mean", "standard_deviation", "percentile")
 
 
-def is_valid_cell_methods_param(cell_methods):
-    """Validate the cell_method parameter supplied by caller"""
-    return cell_methods in VALID_CELL_METHODS_PARAMETERS
+def is_valid_clim_stat_param(climatological_statistic):
+    """Validate the climatological_statistic parameter supplied by caller"""
+    return climatological_statistic in VALID_CLIM_STAT_PARAMETERS
 
 
-def check_final_cell_method(cell_methods, target_method, default_to_mean=True):
+def check_climatological_statistic(
+    cell_methods, climatological_statistic, default_to_mean=True
+):
     """Determines whether the final method in a cell methods string
-    (corresponding to a statistical aggregation) matches the target.
+    (corresponding to a statistical aggregation) matches the target
+    climatological statistic.
```

Comment:
Yeah, this reads quite a bit more clearly, with just the change in terminology. Suggests this was the right naming.